packages/app/package.json

```diff
@@ -10,5 +10,9 @@
       "@types/react-dom": "17.0.11",
       "@ifixit/tsconfig": "workspace:*",
       "typescript": "4.5.3"
+   },
+   "scripts": {
+      "build": "",
+      "clean": ""
```

Comment:
If we forget to add these scripts to new packages in the future, we probably wouldn't realize it until we tried to update react-commerce in the monorepo. I wonder if there's a simple way we can require that these are specified. But maybe that's more work than it's worth.